processors/src/claim-processor/handler.js

```diff
@@ -95,7 +100,7 @@ class ClaimHandler extends TransactionHandler {
     const existingClaim = await state.getClaim(vin);
     if (!existingClaim) {
       logger.error(`Attempted to edit a non-existing claim for VIN: ${vin}.`);
-      throw new InvalidTransaction('Claim does not exist');
+      throw new InvalidTransaction('Cannot edit - claim does not exist');
```

Comment:
Why??????????

Reply:
Just differentiating between the invalid transaction messages for deleting and editing a claim.  I can change it back so they both return the same message if necessary.  Not a big deal.

Reply:
The differentiation isn't necessary because it's implicitly differentiated by whoever calls the api.

Reply:
When I call "edit claim" and get an error, it's obvious that the error happened in edit claim.